src/list.scss

```diff
@@ -0,0 +1,159 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-list;
+
+.#{$block} {
+  @include fd-reset();
+
+  list-style: none;
+  width: 100%;
+
+  &__item,
+  &__group-header,
+  &__footer {
+    @include fd-reset();
+
+    padding: 0 1rem;
+    height: 2.75rem;
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    background: var(--sapList_Background);
+    border-bottom: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+
+    @include fd-hover() {
+      background: var(--sapList_Hover_Background);
+    }
+
+    @include fd-selected() {
+      background: var(--sapList_SelectionBackgroundColor);
+      border-bottom-color: var(--sapList_SelectionBorderColor);
+
+      @include fd-hover() {
+        background: var(--sapList_Hover_SelectionBackground);
+      }
+    }
+
+    @include fd-active() {
+      background: var(--sapList_Active_Background);
+
+      .#{$block}__title,
+      .#{$block}__secondary,
+      .#{$block}__icon {
+        color: var(--sapList_Active_TextColor);
+      }
+    }
+
+    &--no-data {
+      .#{$block}__title {
+        text-align: center;
+      }
+
+      @include fd-hover() {
+        background: var(--sapList_Background);
+      }
+    }
+  }
+
+  &__title,
+  &__secondary {
+    @include fd-reset();
+
+    color: var(--sapList_TextColor);
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  &__title {
+    flex: 3 3 10%;
+    font-size: var(--sapFontLargeSize);
+  }
+
+  &__secondary {
```

Comment:
Question: do we want to call this `status` to align with global design? `fd-list__secondary` makes me think of a sub-list or something... but if you think it needs to be generic to handle other things, no problem.

https://wiki.wdf.sap.corp/wiki/display/visualcore/List+%28Fiori+3%29+-+Standard+List#List(Fiori3)-StandardList-Statusincluded

Reply:
I thought about that as well, however, I wanted to try and keep this more generic (if possible) as there are other examples of this second column of data.

See https://wiki.wdf.sap.corp/wiki/display/visualcore/List+%28Fiori+3%29+-+Item+Counter